### PR TITLE
Add CI

### PR DESCRIPTION
**Pull request by tobolar**
Add stuff for continuous integration, based on MSL

### CODE REVIEW

File: .CI/check_url_access.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+"""
+Copyright (C) 2022, DLR Institut fuer Systemdynamik und Regelungstechnik
+All rights reserved.
+
+Check url adress in links of Modelica HTML documentation for validity
```
Comment (beutlich) [inline .CI/check_url_access.py:5]:
```suggestion
Check url address in links of Modelica HTML documentation for validity
```

File: .CI/check_url_access.py
Comment (beutlich) [inline .CI/check_url_access.py:1]: What was the reason to not use checkLinks.py from MSL (which only uses builtin Py modules and does not depend on bs4)?

Comment (tobolar) [reply .CI/check_url_access.py:1]: Well, to be honest, there is no reasonable cause. I have just wrote the code on my own and noticed later there is a comparable routine in MSL.
Reviewing breefly the two routines, I use BeautifulSoup to evaluate HTML and the code is shorter. But I'm sure I don't cover all cases as `checkLinks.py` does.

Comment (tobolar) [reply .CI/check_url_access.py:1]: Looking deeply, I see the difference (still don't insist on my solution!): `check_url_access.py` handles
- only selected URLError (variable ERR) is printed.
- uses bs4 (which IMO makes the URL extraction easier)
- applies on both '.mo' and '.md' files
- restricts checking on 'http:', 'https:', 'mailto'

Comment (tobolar) [reply .CI/check_url_access.py:1]: @beutlich Ok, I've changed this to MSL routine. Link check runs now on PR (we still have no cron jobs defined)

Comment (tobolar) [reply .CI/check_url_access.py:1]: @beutlich I would merge this PR if you have no more comments.

**Review by beutlich — APPROVED** [review]

**Review by nieweber — APPROVED** [review]

**Pull request merged**